### ISSUE TITLE
rating not working on edit form

### ISSUE THREAD

**Issue by gswartz777** [javiertoledo/bootstrap-rating-input#34]
I have a form where the user can select the number of stars for the rating.  When saving the record and coming back to edit it, the form starts with 5 empty stars rather than the rating that was saved previously.  If I mouse over the stars and then mouse off, the rating then appears properly.  Here's the html...

```
<div class="form-group">
            <label for="company_rating">Rating</label>
            <div class="rating-input">
                   <i class="glyphicon glyphicon-star-empty" data-value="1"></i>
                   <i class="glyphicon glyphicon-star-empty" data-value="2"></i>
                   <i class="glyphicon glyphicon-star-empty" data-value="3"></i>
                   <i class="glyphicon glyphicon-star-empty" data-value="4"></i>
                   <i class="glyphicon glyphicon-star-empty" data-value="5"></i>
            </div>
           <input class="rating hidden" id="company_rating" name="company[rating]" type="number" value="4">
</div>
```

And after mousing over the stars, here's the rendered html.

```
<div class="form-group">
            <label for="company_rating">Rating</label>
            <div class="rating-input">
                   <i class="glyphicon glyphicon-star" data-value="1"></i>
                   <i class="glyphicon glyphicon-star" data-value="2"></i>
                   <i class="glyphicon glyphicon-star" data-value="3"></i>
                   <i class="glyphicon glyphicon-star" data-value="4"></i>
                   <i class="glyphicon glyphicon-star-empty" data-value="5"></i>
           </div>
           <input class="rating hidden" id="company_rating" name="company[rating]" type="number" value="4">
      </div>
```

**Comment by nathansamson** [comment]
Same problem here

**Comment by javiertoledo** [comment]
This has been solved by PR #38
